README.md

```diff
@@ -8,143 +8,107 @@ This repo contains a scaffold to help developers build [apps for Zendesk product
+| [`.babelrc`](#packagejson)              | Configuration file for Babel.js                                                              |
+| [`.browserslistrc`](#packagejson)       | Configuration file for browserslist                                                           |
+| [`jest.config.js`](#packagejson)        | Configuration file for Jest                                                                  |
+| [`package.json`](#packagejson)          | Configuration file for Project metadata, dependencies and build scripts                      |
+| [`postcss.config.js`](#packagejson)     | Configuration file for PostCSS                                                               |
 | [`webpack.config.js`](#webpackconfigjs) | Configuration file that webpack uses to build your app                                       |
 
 #### dist
-The dist directory is the folder you will need to package when submitting your app to the marketplace. It is also the folder you will have to serve when using [ZAT](https://developer.zendesk.com/apps/docs/apps-v2/getting_started#zendesk-app-tools). It includes your app's manifest.json file, an assets folder with all your compiled JavaScript and CSS as well as HTML and images.
-
-#### lib
-The lib directory is where the source code for the app shims and compatibility methods live. While you may modify or remove this code as required for your app, doing so is not recommended for beginners.
+The dist directory is created when you run the app building scripts, it is the folder you will need to package when submitting your app to the marketplace. It is also the folder you will have to serve when using [ZAT](https://developer.zendesk.com/apps/docs/apps-v2/getting_started#zendesk-app-tools). It includes your app's manifest.json file, an assets folder with all your compiled JavaScript and CSS as well as HTML and images.
 
 #### spec
 The spec directory is where all your tests and test helpers live. Tests are not required to submit/upload your app to Zendesk and your test files are not included in your app's package, however it is good practice to write tests to document functionality and prevent bugs.
 
 #### src
-The src directory is where your raw source code lives. The App Scaffold includes different directories for JavaScript, stylesheets, templates and translations. Most of your additions will be in here (and spec, of course!).
+The src directory is where your raw source code lives. The App Scaffold includes different directories for JavaScript, stylesheets, templates, images and translations. Most of your additions will be in here (and spec, of course!).
+
+#### webpack
+In this directory, we have the translations-loader.js which is used by webpack to convert translation json files to javascript objects. We also have the translations-plugin.js to extract multilingual project meta data from translation json files for Zendesk Market.
```

Comment:
This folder contains custom tooling to process translations at build time:
- `translations-loader.js` is used by Webpack to convert `.json` translation files to JavaScript objects, for the app itself.
- `translations-plugin.js` is used to extract compulsory translation strings from the `en.json` file that are used to display metadata about your app on the Zendesk Marketplace.

Reply:
`translations-plugin.js` is used to extract compulsory translation strings from the language json files that are used to display metadata about your app on the Zendesk Marketplace.